drivers/unix/os_unix.cpp

```diff
@@ -74,7 +74,7 @@ static void _setup_clock() {
 	kern_return_t ret = mach_timebase_info(&info);
 	ERR_EXPLAIN("OS CLOCK IS NOT WORKING!");
 	ERR_FAIL_COND(ret != 0);
-	_clock_scale = (double)info.numer / (double)info.denom;
+	_clock_scale = (double)info.numer / (double)info.denom / 1000.0;
```

Comment:
Parentheses would be nice here to make the intent clear.

Reply:
Added

Reply:
I think `(double)info.numer / (double)(info.denom * 1000)` might be better for precision.
as `info.denom` is an integer.